packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -85,11 +87,18 @@ class FrameFlameChart extends CoreElement {
   final _minZoomLevel = 1;
   num _zoomLevel = 1;
 
-  num get _zoomMultiplier => _zoomLevel * 0.075;
+  /// Maximum scroll delta allowed for scrollwheel based zooming.
+  ///
+  // This isn't really needed but is a reasonable for safety in case we
+  // aren't handling some mouse based scroll wheel behavior well, etc.
```

Comment:
nit make these two lines have 3 slashes instead of 2 slashes

Reply:
done.